kensu/google/cloud/bigquery/client.py

```diff
@@ -61,12 +67,28 @@ def query(
                         kensu=kensu,
                         client=client,
                         query=query)
+
                     try:
-                        #TODO: Use sqlparse
-                        index_select = query.lower().index("select")
-                        query_without_insert = query[index_select:]
-                        logger.debug(f"Query without INSERT TO:{query_without_insert}")
+                        if query.lower().replace(" ","").replace("\n","").startswith("insertinto") :
```

Comment:
This is done twice I think

Reply:
yeah could be extracted into a variable of something